integration-tests/gatsby-image/cypress/integration/fluid.js

```diff
@@ -0,0 +1,40 @@
+const fluidTestId = `image-fluid`
+
+describe(`fluid`, () => {
+  beforeEach(() => {
+    cy.visit(`/`)
+  })
+
+  it(`renders a spacer div`, () => {
+    cy.getTestElement(
+      fluidTestId,
+      `.gatsby-image-outer-wrapper > .gatsby-image-wrapper > div`
+    )
+      .should(`have.attr`, `style`)
+      .and(`match`, /width:100%;padding-bottom/)
+  })
+
+  it(`renders sizes`, () => {
+    cy.getTestElement(fluidTestId, `picture > source`).should(
+      `have.attr`,
+      `sizes`,
+      `(max-width: 500px) 100vw, 500px`
+    )
+  })
+  // TODO: make this less janky
+  it(`renders correct srcset`, () => {
```

Comment:
I feel like this could be super flaky/manual, but I haven't found a good way to get "snapshot" behavior in cypress yet. There is a [cypress snapshot](https://github.com/cypress-io/snapshot), but I didn't have the best experience using it.